bin/dry-run.bash

```diff
@@ -21,14 +21,28 @@ if [[ $1 == "sc" ]]; then
   log_info "Running helmfile diff on the service cluster"
   : "${config[config_file_sc]:?Missing service cluster config file}"
 
-  "${here}/ops.bash" helmfile sc diff
+  if [ ${#} -eq 2 ]; then
```

Comment:
I would like that you actually verify that the variable is set to "sync" and not just assuming that it's set correctly.

This applies to all places where this assumption is done.